tests/v1/worker/test_gpu_model_runner.py

```diff
@@ -438,11 +438,14 @@ def test_load_model_weights_inplace(dist_init, model_runner, model_runner_2):
     # model_runner_2 loads dummy weights first then load real weights inplace
     model_runner.load_model()
     original_load_format = model_runner_2.load_config.load_format
-    model_runner_2.load_config.load_format = "dummy"
```

Comment:
should we just have a dedicate test to check `update_config`, or at least check the updated results?

Reply:
Added a dedicated unit test